webdriver/tests/bidi/browsing_context/set_viewport/__init__.py

```diff
@@ -0,0 +1,21 @@
+from webdriver.bidi.modules.script import ContextTarget
+
+async def layout_viewport_dimensions(bidi_session, context):
```

Comment:
So this is not needed anymore given that we have the global fixture for bidi now.